packages/client/src/components/overlay/NotificationDisplay.svelte

```diff
@@ -39,7 +41,8 @@
     display: flex;
     flex-direction: column;
     justify-content: flex-start;
-    align-items: center;
+    align-items: end;
```

Comment:
After testing it out myself, this is the only thing I think we need to change. Aligning notifications to the right looks strange to me, as it even means single notifications aren't actually centered in the screen. I think we should change this back to `center`.

Reply:
Changed back to center align with fixed toast widths.